### DISCUSSION TITLE
Where can I report for a against someone who is using spam code hacking people?

### DISCUSSION THREAD

**Discussion by Khalil15211** [community/community#129285] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

![Screenshot_20240621_114632_com discord](https://github.com/community/community/assets/136367178/665568e7-127f-42e3-9f6e-042123e0881e)
I downloaded a file from github it has infected my pc in a way that hacked my crypto wallet stole assets and now misusing my social account why github allow such users to keep such files on platform been long github didn't remove that file losing trust of platform.

https://github.com/humaphenixo/SE-PROJECT/releases/tag/Download

To github team do install this in your pc you'll know how is he harming people.

**Comment by davevad93** [comment, marked as answer]
Hi @Khalil15211 ,

You can report the repo containing the malicious file, on the right side where the `About` section is located, click on `Report repository`. Select the `Active Malware or Exploits` category and submit your report.

![37fba](https://github.com/community/community/assets/94308234/5111c2f9-14c7-4fa2-92de-dc169d616742)

You can also report a user by clicking on `Block or Report` located under the picture on the main profile page.

![0](https://github.com/community/community/assets/94308234/3a059e38-0135-4a57-b10a-fc8b2c355eff)

**Comment by sellatine** [comment]
hello please message me as i got the same rat and i want to get to the bottom of it
